pymc/pytensorf.py

```diff
@@ -96,12 +97,16 @@ def convert_generator_data(data) -> TensorVariable:
 
 def convert_data(data) -> np.ndarray | Variable:
     ret: np.ndarray | Variable
-    if hasattr(data, "to_numpy") and hasattr(data, "isnull"):
```

Comment:
Should we keep the old logic, but now with an or hasattr(data, is_null) for the second condition?